tests/test_features/test_pca.py

```diff
@@ -0,0 +1,71 @@
+##########################################################################
+## Imports
+##########################################################################
+
+import unittest
+import numpy as np
+import numpy.testing as npt
+
+from tests.dataset import DatasetMixin
+from yellowbrick.features.pca import PCA2D
+
+##########################################################################
+## RadViz Base Tests
+##########################################################################
+
+class PCA2DTests(unittest.TestCase, DatasetMixin):
+
+    def test_scale_t_center_t(self, scale=True, center=True):
```

Comment:
I removed the arguments and added a one line docstrings to the functions. 

I like to use `pytest.raises` to write test cases that should catch thrown exceptions. 

 Any thoughts about adding `pytest` to the requirements.txt?